experimenter/experimenter/targeting/constants.py

```diff
@@ -3726,6 +3726,20 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+FX_146_1_TRAINHOP = NimbusTargetingConfig(
```

Comment:
Note: Naming this is difficult. Very open to suggestions! 

New Tab is targeting Fx145.0.1 users who have the 147.X train-hop.